### DISCUSSION TITLE
Is it possible to import the Nuxt 3 App into another Nuxt App?

### DISCUSSION THREAD

**Discussion by Bohdan909** [nuxt/nuxt#25495] [Questions]
The idea is that in one global Nuxt App we import another encapsulated Nuxt App. Like a module or one component.
It is necessary to insert nuxt app as a widget on the site.

**Comment by angelhdzmultimedia** [comment]
Either https://nuxt.com/docs/guide/going-further/modules
Or https://nuxt.com/docs/getting-started/layers
